packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added a connection timeout to the Imperative REST Client, with a default of 60 seconds. [#2486](https://github.com/zowe/zowe-cli/pull/2486)
+- Enhancement: Added the `SOCKET_CONNECT_TIMEOUT` environment variable to the `EnvironmentalVariableSettings` class to allow extenders to determine how long to wait for a socket connection before timing out. [#2486](https://github.com/zowe/zowe-cli/pull/2486)
```

Comment:
Should we say `ZOWE_SOCKET_CONNECT_TIMEOUT` here? I know the prefix is implied, but users may not be aware of this.

Reply:
I will update it. I was under the impression that Imperative consumers are more extenders, but I agree that is more clear for CLI and VSCE users.